ui/lib/sync/addon/components/secrets/page/overview.ts

```diff
@@ -61,8 +62,22 @@ export default class SyncSecretsDestinationsPageComponent extends Component<Args
     }
   });
 
+  @action
+  clearActivationErrors() {
+    this.activationErrors = null;
+  }
```

Comment:
whoops, looks like i missed this when implementing the sync activation modal. this ensures those errors are cleared once a user re-clicks the "confirm" button in the activation modal.